zio-interop-cats/shared/src/main/scala/zio/interop/catsmtl.scala

```diff
@@ -18,31 +18,13 @@ package zio.interop
 
 import cats.Applicative
 import cats.mtl.*
-import zio.{ CanFail, ZIO, ZTraceElement }
-import zio.internal.stacktracer.{ Tracer => CoreTracer }
+import zio.{ CanFail, ZIO }
 import zio.internal.stacktracer.InteropTracer
 
 abstract class CatsMtlPlatform extends CatsMtlInstances
 
 abstract class CatsMtlInstances {
 
-  implicit def zioLocal[R, E](implicit ev: Applicative[ZIO[R, E, _]]): Local[ZIO[R, E, _], R] =
```

Comment:
@adamgfraser just wondering, why was this and the `Ask` instance removed? can they not be implemented with ZIO 2.x?

Reply:
@lacarvalho91 It can be implemented but needs to build in the `ZEnvironment`. See #662.